### ISSUE TITLE
im using this code with street images

### ISSUE THREAD

**Issue by tabarkarajab** [harveyslash/Facial-Similarity-with-Siamese-Networks-in-Pytorch#31]
im using this code with street image dataset it's just keep on running the  loop, whats the problem ?

folder_dataset_test = dset.ImageFolder(root=Config.testing_dir)
siamese_dataset = SiameseNetworkDataset(imageFolderDataset=folder_dataset_test,
                                        transform=transforms.Compose([transforms.Resize((100,100)),
                                                                      transforms.ToTensor()
                                                                      ])
                                       ,should_invert=False)

test_dataloader = DataLoader(siamese_dataset,num_workers=6,batch_size=1,shuffle=True)
dataiter = iter(test_dataloader)
x0,_,_ = next(dataiter)

for i in range(10):
    _,x1,label2 = next(dataiter)
    concatenated = torch.cat((x0,x1),0)
    
    output1,output2 = net(Variable(x0).cuda(),Variable(x1).cuda())
    euclidean_distance = F.pairwise_distance(output1, output2)
    imshow(torchvision.utils.make_grid(concatenated),'Dissimilarity: {:.2f}'.format(euclidean_distance.item()))

**Comment by harveyslash** [comment]
It will keep on running till a certain number of iterations is reached. You can tweak that value in the loop. This model will take a significant amount of time to run.
